demos/python/flask/run_flask_on_occlum.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+BLUE='\033[1;34m'
+NC='\033[0m'
+
+# Run the python demo
+cd occlum_instance
+echo -e "${BLUE}occlum run /bin/python3 rest_api.py${NC}"
+occlum run /bin/python3 rest_api.py
```

Comment:
I prefer that: Using "shebang" in the rest_api.py and only add the rest_api.py in the trust execution path.
The current implementation is workable but not secure. For example, someone could do this "occlum run /bin/python3 /host/any.py".